test/signin.test.js

```diff
@@ -0,0 +1,63 @@
+const request = require('supertest');
+const expect = require('chai').expect;
+const jwt = require('jsonwebtoken');
+
+const app = require('../index');
+const userFixture = require('../fixtures/user');
+const loadFixture = require('./loadFixture');
+const privateKey = process.env.API_KEY;
```

Comment:
Please rename it `JWT_SECRET` and add a `.env.sample` file with dummy data for all the env variables (basically a copy of `.env` with dummy data for sensitive info).